Convert registry to k8s.gcr.io

thockin opened the pull request:
This PR was auto-generated.  Please apply human expertise to review for correctness.

Followup to https://github.com/kubernetes/kubernetes/pull/54174

xref https://github.com/kubernetes/release/issues/281

k8s-reviewable commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.kubernetes.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/heapster/1919)
<!-- Reviewable:end -->

x13n commented on grafana/RELEASES.md:
```diff
@@ -5,9 +5,9 @@
 
 ## 4.0.2 (04-01-2017)
-- Formalized the image name for every arch to `gcr.io/google_containers/heapster-grafana-ARCH:VERSION`
+- Formalized the image name for every arch to `k8s.gcr.io/heapster-grafana-ARCH:VERSION`
```
Substituting this in old release notes doesn't seem to be the right thing to do.

x13n commented on influxdb/RELEASES.md:
```diff
@@ -4,9 +4,9 @@
 
 ## v0.13.0 (4.1.2016)
-- Formalized the image name for every arch to `gcr.io/google_containers/influxdb-grafana-ARCH:VERSION`
+- Formalized the image name for every arch to `k8s.gcr.io/influxdb-grafana-ARCH:VERSION`
```
ditto

thockin committed:
Convert registry to k8s.gcr.io

thockin commented:
Reverted those parts

On Thu, Dec 28, 2017 at 3:02 AM, Daniel Kłobuszewski <
notifications@github.com> wrote:

> *@x13n* commented on this pull request.
> ------------------------------
>
> In grafana/RELEASES.md
> <https://github.com/kubernetes/heapster/pull/1919#discussion_r158928649>:
>
> > @@ -5,9 +5,9 @@
>  - Support Grafana 4.4.1.
>
>  ## 4.0.2 (04-01-2017)
> -- Formalized the image name for every arch to `gcr.io/google_containers/heapster-grafana-ARCH:VERSION` <http://gcr.io/google_containers/heapster-grafana-ARCH:VERSION>
> +- Formalized the image name for every arch to `k8s.gcr.io/heapster-grafana-ARCH:VERSION` <http://k8s.gcr.io/heapster-grafana-ARCH:VERSION>
>
> Substituting this in old release notes doesn't seem to be the right thing
> to do.
> ------------------------------
>
> In influxdb/RELEASES.md
> <https://github.com/kubernetes/heapster/pull/1919#discussion_r158928671>:
>
> > @@ -4,9 +4,9 @@
>  - Updated to version v1.1.1; bumped Godeps and modified some code in heapster to use the latest schema
>
>  ## v0.13.0 (4.1.2016)
> -- Formalized the image name for every arch to `gcr.io/google_containers/influxdb-grafana-ARCH:VERSION` <http://gcr.io/google_containers/influxdb-grafana-ARCH:VERSION>
> +- Formalized the image name for every arch to `k8s.gcr.io/influxdb-grafana-ARCH:VERSION` <http://k8s.gcr.io/influxdb-grafana-ARCH:VERSION>
>
> ditto
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/heapster/pull/1919#pullrequestreview-85802577>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVBafmfA-TWcfV553pPP7CQ0-jfSBks5tE3VggaJpZM4RLSVH>
> .
>

x13n commented:
/lgtm
/retest
